conbench/app/benchmarks.py

```diff
@@ -337,12 +342,30 @@ def _get_benchmarks(self):
 
 
 rule(
-    "/benchmarks/",
-    view_func=BenchmarkList.as_view("benchmarks"),
+    "/benchmark-results/",
+    view_func=BenchmarkResultList.as_view("benchmark-results"),
     methods=["GET"],
 )
+
+rule(
+    "/benchmark-results/<benchmark_id>/",
+    view_func=BenchmarkResult.as_view("benchmark-result"),
+    methods=["GET", "POST"],
```

Comment:
Ah, one thing that didn't occur to me this morning is that we're posting all results to `/api/benchmarks/`; we should switch those to `/api/benchmark-results/` once we're ready. There are a number of places because our client-side tooling has a lot of variation, but the most obvious place is in benchadapt: https://github.com/conbench/conbench/blob/ab8c64fbd7a88d83d1c84ee74bf0e5bb0d33db5d/benchadapt/python/benchadapt/adapters/_adapter.py#L187 And we should integration test that, which I think we've got a story for somewhere, but we've never gotten around to actually setting up.

Reply:
:+1: noting that this PR and discussion is about the UI routes, i.e. not touching on any URL starting with `/api`.

John also asked about `/api/...` in https://github.com/conbench/conbench/issues/964#issuecomment-1486976661.

Yes, let's also plan to rename the `/api/...` routes with clearer terminology (in which case, I agree, doing the transition rather carefully is even more important).